samples/web/content/apprtc/apprtc.py

```diff
@@ -515,13 +515,18 @@ def post(self, room_id):
     logging.info('Room ' + room_id + ' has state ' + result['room_state'])
 
 class MainPage(webapp2.RequestHandler):
+  def write_response(self, target_page, params={}):
+    template = jinja_environment.get_template(target_page)
+    content = template.render(params)
+    self.response.out.write(content)
+    
   def get(self):
-    """Redirects to a room page."""
-    room_id = generate_random(8)
-    redirect = '/r/' + room_id
-    redirect = append_url_arguments(self.request, redirect)
-    self.redirect(redirect)
-    logging.info('Redirecting visitor to base URL to ' + redirect)
+    """Renders index.html."""
+    # Parse out parameters from request.
+    params = get_room_parameters(self.request, None, None, None)
+    params['connect'] = False
+    params['room_id'] = generate_random(8)
```

Comment:
I'm ok with either way, the intent is clearer this way, but the intent can be made fairly clear with a comments.